components/DetailCountry/index.js

```diff
@@ -35,7 +36,7 @@ export default function DetailCountry({ countries, deleteCountry }) {
   if (currentCountry === undefined) {
     return null;
   }
-  const { name, startDate, endDate, id } = currentCountry;
+  const { name, startDate, endDate, id, path, publicId } = currentCountry;
```

Comment:
the name "path" is a bit unspecific here, it might also mean a physical path to the country, maybe you call it imageUrl / imagePath?